array-filter/main.js

```diff
@@ -0,0 +1,57 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna'
+];
+
+function isEven(num) {
+  let newObject = [];
+  if (num % 2 === 0) {
+    newObject += num;
+    return newObject;
+  }
+}
```

Comment:
Each callback function should return `true` or `false`. So for `isEven` would look like:
```JAVASCRIPT
function isEven(num) {
  if (num % 2 === 0) {
    return true;
  } else {
    return false;
  }
}
```
OR it can be shortened to:
```JAVASCRIPT
function isEven(num) {
  return num % 2 === 0;
}
```